examples/tests/dependency_vmimage.py

```diff
@@ -0,0 +1,44 @@
+import glob
+import os
+
+from avocado import Test
+
+
+class VmimageTest(Test):
+
+    def test(self):
```

Comment:
IIUIC this example doesn't use the new vmimage runner. I am missing the docstring with `:avocado: dependency={"type": "vmimage", "provider": "fedora", "version": "41", "arch": "s390x"}`